Sources/DocCDocumentation/DocCDocumentation.docc/Info.plist

```diff
@@ -22,6 +22,8 @@
 	<string>0.1.0</string>
 	<key>CFBundleVersion</key>
 	<string>0.1.0</string>
+	<key>VersionDisplayName</key>
+	<string>0.1.0</string>
```

Comment:
I think we can hold off on making this change until we actually adopt versioning in DocC's documentation. But definitely something we should look at doing as part of this project in a future PR.